modules/ppcp-blocks/resources/js/checkout-block.js

```diff
@@ -653,19 +664,47 @@ const PayPalComponent = ( {
 	);
 };
 
-const BlockEditorPayPalComponent = () => {
-	const urlParams = {
-		clientId: 'test',
-		...config.scriptData.url_params,
-		dataNamespace: 'ppcp-blocks-editor-paypal-buttons',
-		components: 'buttons',
-	};
+const BlockEditorPayPalComponent = ( { fundingSource, buttonAttributes } ) => {
+	const urlParams = useMemo(
+		() => ( {
+			clientId: 'test',
+			...config.scriptData.url_params,
+			dataNamespace: 'ppcp-blocks-editor-paypal-buttons',
+			components: 'buttons',
+		} ),
+		[]
+	);
+
+	const style = useMemo( () => {
+		const configStyle = normalizeStyleForFundingSource(
+			config.scriptData.button.style,
+			fundingSource
+		);
+
+		if ( buttonAttributes ) {
+			return {
+				...configStyle,
+				height: buttonAttributes.height
+					? Number( buttonAttributes.height )
+					: configStyle.height,
+				borderRadius: buttonAttributes.borderRadius
+					? Number( buttonAttributes.borderRadius )
+					: configStyle.borderRadius,
+				color: buttonAttributes.darkMode ? 'white' : configStyle.color,
```

Comment:
We've removed `darkMode` prop in https://github.com/woocommerce/woocommerce/pull/51109. Please remove it from here too.